src/components/OrderForm.js

```diff
@@ -305,6 +307,21 @@ class OrderForm extends React.Component {
   componentDidMount() {
     this._isMounted = true;
     this.UNSAFE_componentWillReceiveProps(this.props);
+
+    // Recaptcha
+    getRecaptcha()
+      .then(recaptcha => {
+        recaptcha.ready(() => {
+          recaptcha
+            .execute(getRecaptchaSiteKey(), { action: 'OrderForm' })
+            .then(recaptchaToken => {
+              this.setState({ recaptchaToken });
+            });
+        });
+      })
+      .catch(err => {
+        console.log('Recaptcha error', err);
```

Comment:
Can we expose this error to the user at all?

Reply:
An error occuring at this step is either a misconfiguration or a problem with Recaptcha service itself, nothing I would report to the user.

Reply:
Will it block an order from being created?

Reply:
Depends what we do on the backend. If the recaptcha token is mandatory and it’s missing, we might block. Ie: inauthenticités anonymous users.

Reply:
Alright, if we block orders based on a missing token, we should inform the user of an error causing the issue.

Reply:
The idea is that the backend will validate the Order and will send the proper error message. The frontend just have to display the error sent back by the backend (like it's already doing).

If you're interested see the matching work in progress on the backend side:
https://github.com/opencollective/opencollective-api/pull/1518/commits/e5dbbe842b3b97aeaf62ff247a48d1a9d8c595f2